src/Doctrine/DoctrineHelper.php

```diff
@@ -268,4 +307,27 @@ public function isKeyword(string $name): bool
 
         return $connection->getDatabasePlatform()->getReservedKeywordsList()->isKeyword($name);
     }
+
+    /**
+     * this method tries to find the correct MappingDriver for the given namespace/class
+     * To determine which MappingDriver belongs to the class we check the prefixes configured in Doctrine and use the
+     * prefix that has the closest match to the given $namespace.
+     */
+    public function getMappingDriverForNamespace(string $namespace): ?MappingDriver
```

Comment:
Is this method needed? It's used in `MakeEntity`, but it's ultimately only used to determine if "attributes" are being used. Could we re-use `doesClassUseAttributes` for that?

Reply:
good point, i don't know why i created a new method for that...

Reply:
Finally i found the reason i created this function.

In case we create an entity for the first time, the ManagerRegistry does not know this entity yet. Only after the Entity File is saved for the first time, the manager knows it.

To decide which MappingDriver should be used we need somehow the configuration from doctrine. Doctrine offers us to use the `prefix: 'App\Entity'` option to configure whole Namespaces as entities, and exactly for that the function is.